team-norms/retrospectives.md

```diff
@@ -0,0 +1,66 @@
+# Retrospectives
+
+These retrospectives occur every 2 weeks on a Friday. There are 2 hours booked for this purpose. 
```

Comment:
Ongoing worth considering whether loading 2 hours on a Friday afternoon is the best time?

With dojo and team lunch already, as well as trying to leave the week on a clean slate, we've historically been mindful of pushing too much additional extra-curricular in to Fridays